pkg/services/rpcsrv/server_helper_test.go

```diff
@@ -125,6 +130,11 @@ func wrapUnitTestChain(t testing.TB, chain *core.Blockchain, orc OracleHandler,
 	errCh := make(chan error, 2)
 	rpcServer := New(chain, cfg.ApplicationConfiguration.RPC, server, orc, logger, errCh)
 	rpcServer.Start()
+	t.Cleanup(func() {
+		rpcServer.Shutdown()
+		server.Shutdown()
```

Comment:
Move `server.Shutdown()` to a separate commit, it fixes a separate issue.

Reply:
Link related issue to this commit.

Reply:
Shouldnt we add shutdown of the srv:= httptest.NewServer(handler), to a separate commit? but not  server, err := network.NewServer(serverConfig, chain, chain.GetStateSyncModule(), logger), because the function doesn't return?

Reply:
Sure, my bad. It should be http server those shutdown should be in a separate commit.